src/Data/Semigroup/Commutative.purs

```diff
@@ -16,7 +16,7 @@ module Data.Semigroup.Commutative where
 
 import Data.Monoid.Additive (Additive)
 import Data.Monoid.Dual (Dual)
-import Data.Semigroup.Commutative (class Commutative)
```

Comment:
Anybody understand why this line is removed?

Reply:
It's redundant and might become an error in the future? https://github.com/purescript/purescript/issues/3079

Reply:
Ha, I see! I didn't notice that this is the module importing part of itself.